ci/validate_generated_artifacts.sh

```diff
@@ -10,4 +10,4 @@ apt update && apt install -y python3 python-virtualenv unzip
 make
 
 # it's an error if we generated something that wasn't checked in.
-git diff --quiet || exit 1
+git diff --name-only --exit-code || echo "Not all generated files were checked in" || exit 1
```

Comment:
Shouldn't it be `&& exit 1` to make it exit after a successful `echo` command?

Reply:
Sorry I didn't mean to resolve this. We need to verify this line .. yes probably && is needed

Reply:
Good catch!